### PR TITLE
automation: Push echo image to quay at main push

### PR DESCRIPTION
**Pull request by qinqon**
After PRs are merged the echo image has to be builded and pushed to quay
for users to consume it.

### CODE REVIEW

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -22,7 +22,8 @@ jobs:
     needs: go-versions
     env:
       CORE_IMAGE_TAG: main
-      CRI: podman
```
Comment (EdDev) [inline .github/workflows/publish.yaml:25]: I am puzzled on how it worked before then.

File: automation/make.sh [hunk 2]
```diff
@@ -118,7 +122,13 @@ if [ -n "${OPT_ECHO_UNIT_TEST}" ]; then
...
-    ${CRI} build ${ECHO_CHECKUP_PATH} --file ${ECHO_CHECKUP_PATH}/Dockerfile --tag "${full_image_name}"
+    echo "Trying to build image \"${ECHO_IMAGE}\"..."
+    ${CRI} build ${ECHO_CHECKUP_PATH} --file ${ECHO_CHECKUP_PATH}/Dockerfile --tag "${ECHO_IMAGE}"
+fi
+
+if [ -n "${OPT_ECHO_PUSH_IMAGE}" ]; then
+   echo "Pushing \"${ECHO_IMAGE}\"..."
+   ${CRI} push ${ECHO_IMAGE} 
 fi
+
```
Comment (EdDev) [inline automation/make.sh:133]: Can you remove these tail newlines?

**Review by EdDev — COMMENTED** [review]
Thank you!
Just a few nits.

File: .github/workflows/publish.yaml (hunk 1 above)
Comment (qinqon) [reply .github/workflows/publish.yaml:25]: That's an error I changed for testing the `act` ubuntu image does not contain podman

**Review by EdDev — APPROVED** [review]
Thanks!

**Pull request merged**